litmus-portal/frontend/src/components/WorkflowCluster/index.tsx

```diff
@@ -26,10 +25,8 @@ const WorkflowCluster = () => {
   const classes = useStyles();
   const [value, setValue] = React.useState('Experiment');
 
-  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setValue((event.target as HTMLInputElement).value);
-  };
-  const handleClick = () => {};
+  const handleChange = () => {};
+  const handleClick=()=>{}
```

Comment:
why does these two lines have different formatting...are you not running prettier?

Reply:
prettier it is not working in installed it, Can you help me also npm run format showing err